applications/asset_tracker_v2/sample.yaml

```diff
@@ -163,7 +163,7 @@ tests:
       - thingy91_nrf9160_ns
     extra_args: >
       EXTRA_CONF_FILE="overlay-lwm2m.conf;overlay-debug.conf"
-      SNIPPET="nrf91-modem-trace-uart"
+      SNIPPET="nrf91-modem-trace-uart;tfm-enable-share-uart"
```

Comment:
use `required_snippets` instead, e.g.:
```
required_snippets:
 - nrf91-modem-trace-uart
 - tfm-enable-share-uart
```

Reply:
This will need a fromtree in sdk-zephyr, I will post one

Reply:
Thank you, I did not know this was possible to add the snippets in that way!